db/write_batch.cc

```diff
@@ -22,12 +22,12 @@
 //    kTypeColumnFamilySingleDeletion varint32 varstring
 //    kTypeColumnFamilyRangeDeletion varint32 varstring varstring
 //    kTypeColumnFamilyMerge varint32 varstring varstring
-//    kTypeBeginPrepareXID varstring
-//    kTypeEndPrepareXID
+//    kTypeBeginPrepareXID
+//    kTypeEndPrepareXID varstring
```

Comment:
Could you add an entry for `kTypeCommitXIDAndTimestamp varstring varstring`? Thanks!

Reply:
Sure, done.